### PR TITLE
add later versions of packages 

### PR DESCRIPTION
**Pull request by LinaMuryanto**
add later versions of packages for of py-deepdiff py-mysql-connector-python py-nbconvert py-pyqt5-sip py-pyqt5 py-scikit-image py-sqlalchemy py-tabulate py-texttable py-typing-extensions

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-mysql-connector-python/package.py
Comment (haampie) [inline var/spack/repos/builtin/packages/py-mysql-connector-python/package.py:1]: protobuf no longer a dependency

File: var/spack/repos/builtin/packages/py-pyqt5/package.py
Comment (haampie) [inline var/spack/repos/builtin/packages/py-pyqt5/package.py:1]: ```depends_on("py-pyqt5-sip@12.13:12", when="@5.15.10:", type=("build", "run"))```

File: var/spack/repos/builtin/packages/py-scikit-image/package.py
Comment (haampie) [inline var/spack/repos/builtin/packages/py-scikit-image/package.py:1]: ```py-imageio@2.27:```
```py-lazy-loader@0.2:```
```py-packaging@21:```
```py-tifffile@2022.8.12:```

File: var/spack/repos/builtin/packages/py-sqlalchemy/package.py
Comment (haampie) [inline var/spack/repos/builtin/packages/py-sqlalchemy/package.py:1]: ```py-typing-extensions@4.6:```

**Review by haampie — CHANGES_REQUESTED** [review]
Needs updates to bounds in deps

File: var/spack/repos/builtin/packages/py-mysql-connector-python/package.py
Comment (LinaMuryanto) [reply var/spack/repos/builtin/packages/py-mysql-connector-python/package.py:1]: @hammpie do I just drop the depends_on() line for protobuf?

File: var/spack/repos/builtin/packages/py-sqlalchemy/package.py
Comment (LinaMuryanto) [reply var/spack/repos/builtin/packages/py-sqlalchemy/package.py:1]: @haampie same for this one. I am guessing that I need to put the 'when' attribute. correct?

File: var/spack/repos/builtin/packages/py-scikit-image/package.py
Comment (LinaMuryanto) [reply var/spack/repos/builtin/packages/py-scikit-image/package.py:1]: @haampie do I need to add 'when' clause for the above? if so, could you tell me how to find out the value for the when clause? thanks.

**Comment by LinaMuryanto** [comment]
I am closing it. I do not know enough about these packages to make the changes.

**Pull request closed without merging**